syncd/FlexCounter.cpp

```diff
@@ -1495,7 +1516,7 @@ class CounterContext : public BaseCounterContext
                     }
 
                     StatType counter = static_cast<StatType>(statCapability.stat_enum);
-                    m_supportedCounters.insert(counter);
+                    supportedCounters.insert(counter);
```

Comment:
why adding parameter, you have passed member here, so not better to use member directly ?

Reply:
Please see above reply